includes/wf_crm_webform_preprocess.inc

```diff
@@ -474,6 +474,23 @@ class wf_crm_webform_preprocess extends wf_crm_webform_base {
               webform_component_update($component);
             }
             $element['#options'] = $new;
+            // Handle optgroups.
```

Comment:
Hey @vinuvarshith , 
please can you write some descriptive comments here or perhaps just before each if statements to give an idea of the scenarios that each if statements handles?
Because "Handle optogroup" is very abstract.
Thanks :)